packages/flutter/lib/src/material/floating_action_button.dart

```diff
@@ -44,16 +44,23 @@ class FloatingActionButton extends StatefulWidget {
     this.mini: false
   }) : super(key: key);
 
+  /// The widget below this widget in the tree.
   final Widget child;
+
   final String tooltip;
+
+  /// The color to paint behind the child.
```

Comment:
that's not really what this is. It's the colour with which to fill the circle / button.